core/src/main/java/com/linecorp/armeria/server/ServiceBuilder.java

```diff
@@ -0,0 +1,123 @@
+package com.linecorp.armeria.server;
```

Comment:
If you rebase your branch against `master` you'll start to see many checkstyle violations, like missing license headers.